apps/console/src/styles/app.css

```
@@ -0,0 +1,313 @@
+  min-height: 56px;
+}
+.topbar-title { display: flex; flex-direction: column; gap: 1px; flex: 0 0 auto; min-width: 0; }
+.topbar-crumb {
+  font-family: 'JetBrains Mono', monospace;
+  font-size: 10px; font-weight: 500;
+  letter-spacing: 0.10em; text-transform: uppercase;
+  color: var(--app-fg-muted);
+}
+.topbar-h1 {
+  margin: 0;
+  font-family: 'Lora', Georgia, serif;
```

Stylelint (17.12.0)

[error] Expected no quotes around "Lora" (font-family-name-quotes)

(font-family-name-quotes)